lib/browser/api/menu-item-roles.js

```diff
@@ -0,0 +1,146 @@
+const roles = {
+  about: {
+    get label () {
+      const {app} = require('electron')
+      return `About ${app.getName()}`
+    }
+  },
+  close: {
+    label: 'Close',
+    accelerator: 'CmdOrCtrl+W',
+    windowMethod: 'close'
+  },
+  copy: {
+    label: 'Copy',
+    accelerator: 'CmdOrCtrl+C',
+    webContentsMethod: 'copy'
+  },
+  cut: {
+    label: 'Cut',
+    accelerator: 'CmdOrCtrl+X',
+    webContentsMethod: 'cut'
+  },
+  delete: {
+    label: 'Delete',
+    accelerator: 'Delete',
+    webContentsMethod: 'delete'
+  },
+  front: {
+    label: 'Bring All to Front'
+  },
+  help: {
+    label: 'Help'
+  },
+  hide: {
+    get label () {
+      const {app} = require('electron')
+      return `Hide ${app.getName()}`
+    },
+    accelerator: 'Command+H'
+  },
+  hideothers: {
+    label: 'Hide Others',
+    accelerator: 'Command+Alt+H'
+  },
+  minimize: {
+    label: 'Minimize',
+    accelerator: 'CmdOrCtrl+M',
+    windowMethod: 'minimize'
+  },
+  paste: {
+    label: 'Paste',
+    accelerator: 'CmdOrCtrl+V',
+    webContentsMethod: 'paste'
+  },
+  pasteandmatchstyle: {
+    label: 'Paste and Match Style',
+    accelerator: 'Shift+Command+V',
+    webContentsMethod: 'pasteAndMatchStyle'
+  },
+  quit: {
+    get label () {
+      const {app} = require('electron')
+      return process.platform === 'win32' ? 'Exit' : `Quit ${app.getName()}`
```

Comment:
It's `Quit` (without app name) on Linux.

https://github.com/sindresorhus/anatine/blob/8a091bbd62057b373904da606223882547f2b491/menu.js#L324